src/OrchardCore.Modules/OrchardCore.Email.Azure/Services/AzureEmailProviderBase.cs

```diff
@@ -224,4 +226,21 @@ private EmailMessage FromMailMessage(MailMessage message, Dictionary<string, ILi
 
         return emailMessage;
     }
+
+    private static EmailAddress ConvertEmailAddressToAzureEmailAddress(string emailWithDisplayName)
+    {
+        var (displayName, emailAddress) = TryCreateMailAddressOrFail(emailWithDisplayName);
+
+        return new EmailAddress(emailAddress, displayName);
+    }
+
+    private static (string DisplayName, string EmailAddress) TryCreateMailAddressOrFail(string email)
+    {
+        if (MailAddress.TryCreate(email, out var mailAddress))
```

Comment:
You can use `IEmailAddressValidator` instead

Reply:
I need the deconstructed email address (i.e. the display name and the address itself separately), not just validation.